### PR TITLE
Customize "Team" name

### PR DESCRIPTION
**Pull request by themsaid**
This PR allows setting a customized name for the team:

```
public function register()
{
    Spark::setTeamString('band');
}
```

This will results all urls to be changed to `band` and `bands`, also all instances of team inside the UI will be changed to `band` and `bands`.

Note: Pluralisation is done via `str_plural()`.

### CODE REVIEW

File: src/Http/routes.php [hunk 1]
```diff
@@ -33,61 +33,61 @@
     // Teams...
     if (Spark::usesTeams()) {
         // General Settings...
-        $router->get('/settings/teams/roles', 'Settings\Teams\TeamMemberRoleController@all');
-        $router->get('/settings/teams/{team}', 'Settings\Teams\DashboardController@show');
+        $router->get('/settings/'.str_plural(Spark::teamString()).'/roles', 'Settings\Teams\TeamMemberRoleController@all');
```
Comment (taylorotwell) [inline src/Http/routes.php]: In this file you could pluralize it once and use same variable after that to save on performance.

Comment (themsaid) [reply src/Http/routes.php]: 👍

**Comment by trouttdev** [comment]
Would it be possible to also have the routes named? This would be nice for use in views and redirects.

**Comment by themsaid** [comment]
@mltroutt well that would be a different PR, but yeah we have this in our feature requests list :)

**Comment by taylorotwell** [comment]
This was all merged.

**Pull request closed without merging**